src/components/authentication/LoginForm.vue

```diff
@@ -0,0 +1,125 @@
+<script setup>
+import { useForm } from 'vee-validate'
+import * as yup from 'yup'
+import { ref, computed } from 'vue'
+import { useUserStore } from '../../stores/userStore'
+
+const schema = yup.object({
+  email: yup.string().email().required(),
+  password: yup.string().min(6).required()
+})
+
+const { defineField, errors, handleSubmit } = useForm({
+  validationSchema: schema
+})
+
+const [email, emailAttrs] = defineField('email', {
+  validateOnModelUpdate: false
+})
+const [password, passwordAttrs] = defineField('password', {
+  validateOnModelUpdate: false
+})
+
+const onSubmit = handleSubmit((values) => {
+  const userStore = useUserStore()
+  const { email, password } = values
+  const result = userStore.login(email, password)
+  result ? emit('success') : emit('failure')
+})
+
+const emit = defineEmits(['success', 'failure', 'changeModal', 'inputStart'])
+</script>
+
+<template>
+  <div class="autho-modal__content login-container">
```

Comment:
typo? auth-modal instead of autho-modal for the block part

Reply:
also, I understand that the auth-modal__content element is a child the body/main element from the page. If it's inside main, please use main__login-container instead, if it is in the root directly, it can have only its block class name

Reply:
Auth-modal__content is the child of auth__modal. That one is kind of by principle an orphan that's generated outside the body